config/locales/en.yml

```diff
@@ -1,33 +1,21 @@
-# Files in the config/locales directory are used for internationalization
-# and are automatically loaded by Rails. If you want to use locales other
-# than English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t "hello"
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t("hello") %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# The following keys must be escaped otherwise they will not be retrieved by
-# the default I18n backend:
-#
-# true, false, on, off, yes, no
-#
-# Instead, surround them with single quotes.
-#
-# en:
-#   "true": "foo"
-#
-# To learn more, please read the Rails Internationalization guide
-# available at https://guides.rubyonrails.org/i18n.html.
-
 en:
-  hello: "Hello world"
+  hello: 'Hello world'
+  title: 'Title'
+  log: 'LOG'
+  goals: 'GOALS'
+  tasks: 'TASKS'
+  profile: 'PROFILE'
+  new_entry: 'Add New Log Entry'
+  create_entry: 'Create New Entry'
+  welcome: 'Welcome to journey!'
+  journey: 'Journey.'
+
```

Comment:
Let's remove extra lines. The values work without quotes as well.
Let's study about some ways to format the translation files, so that the error messages are displayed according to the models.
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models